.github/workflows-source/ci-main.yml

```diff
@@ -246,13 +246,7 @@ jobs:
         shell: bash
         run: |
           set -euo pipefail
```

Comment:
Does this still make sense if we execute a single command? You could consider just having:
```yaml
run: "$CI_PROJECT_DIR"/ci/scripts/run-build-ic.sh
```